src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,54 @@
 package core.basesyntax.impl;
 
+import core.basesyntax.Pair;
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUM_VALUE = 10;
```

Comment:
variable name is unclear to understand - what max number value? we have int numbers until 2147483647 
```suggestion
    private static final int MAX_STORAGE_SIZE = 10;
```